examples/rest/stocks-ws.py

```diff
@@ -0,0 +1,29 @@
+from polygon import WebSocketClient
```

Comment:
Should this and the other websocket demo script live in `examples/websocket` instead of `examples/rest`?